<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScientistDialog</class>
 <widget class="QDialog" name="AddScientistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add scientist</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_add_scientist">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Add a new scientist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_scientist_FirstName">
             <property name="text">
              <string>First Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_error_scientist_first_name">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="input_scientist_first_name"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_scientist_last_name">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Last Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_error_scientist_last_name">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="input_scientist_last_name">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_scientist_gender">
           <property name="text">
            <string>Gender:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="input_scientist_gender">
       <item>
        <property name="text">
         <string>female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>male</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_scientist_year_born">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Year of birth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_error_scientist_year_born">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_scientist_year_born"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_scientist_is_dead">
     <property name="text">
      <string>Is the scientist dead?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_scientist_year_death">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Year of death:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_scientist_year_of_death">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_scientist_year_death">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_add_scientist">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_add_scientist</zorder>
  <zorder>input_scientist_year_born</zorder>
  <zorder>button_add_scientist</zorder>
  <zorder>input_scientist_year_death</zorder>
  <zorder>checkbox_scientist_is_dead</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
